pg_analytics/tests/udf.rs

```diff
@@ -85,6 +88,7 @@ fn udf_deletion(mut conn: PgConnection) {
 }
 
 #[rstest]
+#[ignore]
```

Comment:
It might be helpful to leave a reason for why we're ignoring this test: `#[ignore = "reason"]`